linux/base.Dockerfile

```diff
@@ -135,8 +135,17 @@ RUN bash ./tdnfinstall.sh \
   gh \
   redis \
   cpio \
-  gettext \
-  terraform
+  gettext
+
+# Get latest version of Terraform
+RUN TF_VERSION=$(curl -s https://checkpoint-api.hashicorp.com/v1/check/terraform | jq -r -M ".current_version") \
```

Comment:
Are we trying to not use MAriner terraform? If there is a security vulnerability, in Mariner terraform we should handle this differently.

Reply:
Terraform update is part of routine procedure after discussing with Mariner. @darrentu : can you add this comment just to keep track why we chose to deviate from Mariner's path.